src/thrift_parser.yrl

```diff
@@ -190,8 +190,8 @@ BaseType -> double: double.
 BaseType -> string: string.
 BaseType -> binary: binary.
 
-MapType -> map '<' FieldType ',' FieldType '>': {'$3', '$5'}.
-SetType -> set '<' FieldType '>': '$3'.
+MapType -> map '<' FieldType Annotations ',' FieldType Annotations '>': {'$3', '$6'}.
+SetType -> set '<' FieldType Annotations '>': '$3'.
```

Comment:
Added these to match `ListType`/`Field`, it seems that annotations can follow field type every where else.